src/main/java/com/spring/board/springboard/post/domain/Post.java

```diff
@@ -65,17 +65,19 @@ public Member getMember() {
         return member;
     }
 
-    public void change(String newTitle, String newContent){
-        changeTitle(newTitle);
-        changeContent(newContent);
+    public void change(String newTitle, String newContent) {
+        Assert.notNull(newTitle, "제목을 입력해주세요");
+        Assert.notNull(newContent, "내용을 입력해주세요.");
```

Comment:
```suggestion
	validate(newTitle, newContent, this.member, this.createdAt);
```
생성과 변경에 대한 검증이 동일하다면 이렇게 작성할 수 있을 것 같아요. 검증 정책이 변경될 때 실수를 줄일 수 있을 것 같네요